src/plugins/discover/public/application/components/sidebar/discover_field_details.tsx

```diff
@@ -41,62 +41,66 @@ export function DiscoverFieldDetails({
   const warnings = getWarnings(field);
 
   return (
-    <div className="dscFieldDetails">
-      {!details.error && (
-        <EuiText size="xs">
-          <FormattedMessage
-            id="discover.fieldChooser.detailViews.topValuesInRecordsDescription"
-            defaultMessage="Top 5 values in"
-          />{' '}
-          {!indexPattern.metaFields.includes(field.name) && !field.scripted ? (
-            <EuiLink onClick={() => onAddFilter('_exists_', field.name, '+')}>
-              {details.exists}
-            </EuiLink>
-          ) : (
-            <span>{details.exists}</span>
-          )}{' '}
-          / {details.total}{' '}
-          <FormattedMessage
-            id="discover.fieldChooser.detailViews.recordsText"
-            defaultMessage="records"
-          />
-        </EuiText>
-      )}
-      {details.error && <EuiText size="xs">{details.error}</EuiText>}
-      {!details.error && (
-        <div style={{ marginTop: '4px' }}>
-          {details.buckets.map((bucket: Bucket, idx: number) => (
-            <DiscoverFieldBucket
-              key={`bucket${idx}`}
-              bucket={bucket}
-              field={field}
-              onAddFilter={onAddFilter}
-            />
-          ))}
-        </div>
-      )}
+    <>
+      <div className="dscFieldDetails">
+        {details.error && <EuiText size="xs">{details.error}</EuiText>}
+        {!details.error && (
+          <div style={{ marginTop: '4px' }}>
+            {details.buckets.map((bucket: Bucket, idx: number) => (
+              <DiscoverFieldBucket
+                key={`bucket${idx}`}
+                bucket={bucket}
+                field={field}
+                onAddFilter={onAddFilter}
+              />
+            ))}
+          </div>
+        )}
 
-      {details.visualizeUrl && (
-        <>
-          <EuiLink
-            onClick={() => {
-              getServices().core.application.navigateToApp(details.visualizeUrl.app, {
-                path: details.visualizeUrl.path,
-              });
-            }}
-            className="kuiButton kuiButton--secondary kuiButton--small kuiVerticalRhythmSmall"
-            data-test-subj={`fieldVisualize-${field.name}`}
-          >
-            <FormattedMessage
-              id="discover.fieldChooser.detailViews.visualizeLinkText"
-              defaultMessage="Visualize"
-            />
+        {details.visualizeUrl && (
+          <>
+            <EuiLink
+              onClick={() => {
+                getServices().core.application.navigateToApp(details.visualizeUrl.app, {
+                  path: details.visualizeUrl.path,
+                });
+              }}
+              className="kuiButton kuiButton--secondary kuiButton--small kuiVerticalRhythmSmall"
```

Comment:
This is an interesting mesh of using an EUI component overridden by KUI styles 😆 

I would remove all these classes and depend on the type of EUI component you want it to render as.

Reply:
Done but still had to use some CSS as the button needs to be smaller than the ones available on Eui.